client/src/component/NavBar/NavBar.jsx

```diff
@@ -1,32 +1,72 @@
-import React, {useState, useEffect} from "react";
-import {useSelector} from 'react-redux'
+import React from "react";
+import { useSelector } from "react-redux";
+import { theStore } from "../../index";
 
-import { Link, Nav, Ul } from "../Styles/navStyles";
+import { Link, Nav, Ul, Button } from "../Styles/navStyles";
 
 const NavBar = () => {
   const user = useSelector(state => state.user);
 
+  const handleLogout = async e => {
+    // await Axios.post(`${endpoint}/logout`);
+    theStore.dispatch({
+      type: "user logged out"
+    });
+    window.location.replace("/");
+  };
+
   return (
     <Nav>
-      <Link exact to="/" className="brand-logo" style={{backgroundColor: "unset"}}>
+      <Link
+        exact
+        to="/"
+        className="brand-logo"
+        style={{ backgroundColor: "unset" }}
+      >
+        
         codetrain
       </Link>
-        {
-        (user.user_types_id === 1) ? 
-      <></> : (user.user_types_id === 2) ?
-      <></> : (user.user_types_id === 3) ? 
-      <></> :
-          <Ul>
-        <li>
-          <Link to="/signup">Sign up</Link>
-        </li>
-        <li>
-          <Link to="/login">Log in</Link>
-        </li>
-      </Ul>
-    }
-    </Nav> 
-
+        <Button>Resources</Button>
+      {user.user_types_id === 1 && user.auth === true ? (
+        //  Admin User NavBar
+        <Ul>
+          <Button onClick={handleLogout}>Logout</Button>
+        </Ul>
+      ) : user.user_types_id === 2 && user.auth === true ? (
+        //  BootCamp User NavBar
+        <Ul>
+          <Button to="/dashboard">Dashboard</Button>
+          <Button>Profile</Button>
+          <Button>Jobs</Button>
+          <Button>Applicants</Button>
+          <Button onClick={handleLogout}>Logout</Button>
+        </Ul>
+      ) : user.user_types_id === 3 && user.auth === true ? (
```

Comment:
we may need to revisit auth on companies. But this is fine for now. If we have to auth type 2 usrs and type 3, could be a lot?

Reply:
yeah as I've mentioned before it becomes alot to check if a user actually belongs to said company so that he can post on behalf of them. our site admin has alot of work to do. I hope we pay him/her well